Makefile

```diff
@@ -106,7 +106,7 @@ verify-commits:
 #
 # Example:
 #   make update
-update: build
+update:
```

Comment:
because compilation often depends on generated files. For verify it's ok, but updating should be possible without having a buildable state.